packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts

```diff
@@ -87,23 +87,30 @@ export class ImportCustomEmojisProcessorService {
 				await this.emojisRepository.delete({
 					name: emojiInfo.name,
 				});
-				const driveFile = await this.driveService.addFile({
-					user: null,
-					path: emojiPath,
-					name: record.fileName,
-					force: true,
-				});
-				await this.customEmojiService.add({
-					name: emojiInfo.name,
-					category: emojiInfo.category,
-					host: null,
-					aliases: emojiInfo.aliases,
-					driveFile,
-					license: emojiInfo.license,
-					isSensitive: emojiInfo.isSensitive,
-					localOnly: emojiInfo.localOnly,
-					roleIdsThatCanBeUsedThisEmojiAsReaction: [],
-				});
+				try {
+					const driveFile = await this.driveService.addFile({
+						user: null,
+						path: emojiPath,
+						name: record.fileName,
+						force: true,
+					});
+  				await this.customEmojiService.add({
+  					name: emojiInfo.name,
+  					category: emojiInfo.category,
+  					host: null,
+  					aliases: emojiInfo.aliases,
+  					driveFile,
+  					license: emojiInfo.license,
+  					isSensitive: emojiInfo.isSensitive,
+  					localOnly: emojiInfo.localOnly,
+  					roleIdsThatCanBeUsedThisEmojiAsReaction: [],
+  				});
```

Comment:
```suggestion
  					await this.customEmojiService.add({
  						name: emojiInfo.name,
	  					category: emojiInfo.category,
	  					host: null,
  						aliases: emojiInfo.aliases,
  						driveFile,
	  					license: emojiInfo.license,
  						isSensitive: emojiInfo.isSensitive,
  						localOnly: emojiInfo.localOnly,
  						roleIdsThatCanBeUsedThisEmojiAsReaction: [],
  					});
```

The indentation needs to be raised one level

Reply:
also Misskey uses only TABs